db-scripts/src/main/resources/migration.sql

```diff
@@ -588,3 +591,15 @@ UPDATE `info` SET `DB_SCHEMA_VERSION`="2.8.1";
 ALTER TABLE `mutation_event` DROP KEY `KEY_MUTATION_EVENT_DETAILS`;
 ALTER TABLE `mutation_event` ADD KEY `KEY_MUTATION_EVENT_DETAILS` (`CHR`, `START_POSITION`, `END_POSITION`, `TUMOR_SEQ_ALLELE`(240), `ENTREZ_GENE_ID`, `PROTEIN_CHANGE`, `MUTATION_TYPE`);
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.8.2";
+
+##version: 2.9.0
+ALTER TABLE `gene` DROP COLUMN `CYTOBAND`, DROP COLUMN `LENGTH`;
+ALTER TABLE `cancer_study` ADD COLUMN `REFERENCE_GENOME_ID` INT(4) DEFAULT 1,
+                           ADD CONSTRAINT `FK_REFERENCE_GENOME` FOREIGN KEY (`REFERENCE_GENOME_ID`)
+                               REFERENCES `reference_genome`(`REFERENCE_GENOME_ID`) ON DELETE CASCADE;
+UPDATE `cancer_study`
+    INNER JOIN `genetic_profile` ON `cancer_study`.CANCER_STUDY_ID = `genetic_profile`.CANCER_STUDY_ID
+    INNER JOIN `mutation` ON `mutation`.GENETIC_PROFILE_ID = `genetic_profile`.GENETIC_PROFILE_ID
+    INNER JOIN `mutation_event` ON `mutation`.MUTATION_EVENT_ID = `mutation_event`.MUTATION_EVENT_ID
+SET `cancer_study`.REFERENCE_GENOME_ID = IF(`mutation_event`.NCBI_BUILD in (37, 'hg19','GRCh37'), 1, 2);
```

Comment:
`mutation_event.NCBI_BUILD` is `VARCHAR(10)` so shouldn't `37` be in quotes also?

Reply:
right, should be. Good catch!

Reply:
this is not yet fixed.
Also .. here we are assuming that reference genome id "1" is for hg19 and reference genome id "2" is for <anything else>. Above in the migration script we added hg19 and hg38. So this migration step may do the wrong thing if a portal installation has other builds in the database (hg18?)

Reply:
quoted 37.
if you look at migration script line 399~402, reference_genome_ids are manually assigned. hg19 is always 1, hg38 is always 2. 
We only have a handful reference genomes. If anything goes wrong, we can always fix manually and update your migration.sql.